### ISSUE TITLE
Service: googlewifi.reset does not restart LAN interface

### ISSUE THREAD

**Issue by nathang21** [djtimca/hagooglewifi#81]
tl;dr this call appears to only reset the WiFi interface, but not the LAN interface. 

All my wired devices remain connected to the router throughout the reset, which I have verified by ping. Even external internet requests while hard wired - continue to work. I realize there may not be anything this integration can do, as it's just calling the API's provided by Google, but I wanted to start an issue to document this for others who may misunderstand what this reset actually does. 

--- 

I discovered this while debugging issues between my modem <=> Router, and to mitigate i'm trying to restart both nightly in sequence, which I have been doing manually for a while. 

I'm using a Zigbee smart outlet for my modem, and was hoping to use this software restart, for the router. However it appears this does not actually restart the host OS on the routers, but perhaps just the WiFi interface/driver/mesh network.

To accomplish a full hardware restart, I may also need a smart outlet for the primary google router, will follow-up here once I know more.

Automation i'm using:
```
alias: Reset Internet Nightly
description: Modem + Google WiFi
trigger:
  - platform: time
    at: "04:30:00"
condition: []
action:
  - service: googlewifi.reset
    data:
      entity_id: binary_sensor.google_wifi_system_<UUID>
  - delay:
      hours: 0
      minutes: 15
      seconds: 0
      milliseconds: 0
  - service: switch.turn_off
    data: {}
    target:
      device_id: <UUID>
  - delay:
      hours: 0
      minutes: 0
      seconds: 30
      milliseconds: 0
  - service: switch.turn_on
    data: {}
    target:
      device_id: <UUID>
mode: single
```

**Comment by nathang21** [comment]
Perhaps I was wrong, I am seeing a short almost 30 second window where the wired connection to the router is down, where as local connectivity to wireless devices within the mesh takes about 90 seconds. 
![image](https://user-images.githubusercontent.com/7875110/192150196-ae68493c-9941-4f91-8c05-5bf424e1f129.png)

When I originally discussed the issue (lack of gap in ping) I wasn't restarting the modem, but perhaps that was a fluke, as the modem should not impact the LAN downstream of the router at all.
